kairon/chat/handlers/channels/clients/whatsapp/gupshup.py

```
@@ -0,0 +1,274 @@
+            return False, last_status_code, last_response
+        except ClientResponseError as cre:
+            return False, last_status_code, {"error": str(cre), "response": last_response}
+        except ClientConnectionError as cce:
+            return False, last_status_code, {"error": str(cce), "response": last_response}
+        except ClientError as ce:
+            return False, last_status_code, {"error": str(ce), "response": last_response}
+        except Exception as e:
+            return False, last_status_code, {"error": str(e), "response": last_response}
+
+    async def send_template_message_async(self, name: str, to_phone_number: str, language_code: str = "en",
+                                          components: dict = None, namespace: str = None) -> (bool, int, any):
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)